README.md

```diff
@@ -49,6 +49,15 @@ The extension can be activated in two ways
 
 Originally syntax from the [Confluence® Textmate Bundle](https://github.com/alkemist/Confluence.tmbundle).
 
+## Development
```

Comment:
Hi @NSHenry I would prefer to extract a development block to separate file, couse 'README' is a plugin-description page, I think it will be better to create a separate file, smth like `DEVELOPMENT_README` or what ever.

Reply:
No worries. I'll get rid of these comments. They were more of a personal note.